README.md

```diff
@@ -1,101 +1,103 @@
+- [18-370: Fundamentals of Control](electives/18370.md)
+- [18-421/623: Analog Integrated Circuit Design](electives/18623.md)
+- [18-429/729: Board-Level RF Systems for the Internet-of-Things](electives/18729.md)
+- [18-447: Introduction to Computer Architecture](electives/18447.md)
+- [18-491: Digital Signal Processing](electives/18491.md)
+- [18-540/745: Rapid Prototyping of Computer Systems](electives/18540.md)
+- [18-578: Mechatronic Design](electives/18578.md)
+- [18-640: Hardware Arithmetic for Machine Learning](electives/18640.md)
+- [18-652: Foundations of Software Engineering](electives/18652.md)
+- [18-660: Optimization](electives/18660.md)
+- [18-661: Introduction to Machine Learning for Engineers](electives/18661.md)
+- [18-665: Advanced Probability & Statistics for Engineers](electives/18665.md)
+- [18-690: Introduction to Neuroscience for Engineers](electives/18690.md)
+- [18-698: Neural Signal Processing](electives/18698.md)
+- [18-723: RF IC Design and Implementation](electives/18723.md)
+- [18-740: Modern Computer Architecture and Design](electives/18740.md)
+- [18-741: Computer Networks](electives/18741.md)
+- [18-746: Storage Systems](electives/18746.md)
+- [18-747: How to Write Low Power Code for IoT](electives/18747.md)
+- [18-749: Building Reliable Distributed Systems](electives/18749.md)
+- [18-759: RW Wireless Networks](electives/18759.md)
+- [18-785: Data Inference and Applied Machine Learning](electives/18785.md)
+- [18-792: Advanced Digital Signal Processing](electives/18792.md)
+- [18-793: Image and Video Processing](electives/18793.md)
+- [18-794: Pattern Recognition Theory](electives/18794.md)
+- [15-821/18-843: Mobile and Pervasive Computing](electives/18843.md)
+- [18-847C: Data Center Computing](electives/18847.md)
+- [18-847F: Foundations of Cloud and Machine Learning Infrastructure](electives/18847F.md)
+- [18-898D: Graph Signal Processing and Geometric Learning](electives/18898D.md)
+- [24-104: Maker Series: Intro to Modern Making](electives/24104.md)
+- [80-180: Nature of Language](electives/80180.md)
+- [80-405: Game Theory](electives/80405.md)
+- [82-137: Chinese Calligraphy: Culture and Skils](electives/82137.md)
+- [82-208: Eastern Europe: Society and Culture](electives/82208.md)
+
+## Research Papers
+
+- [Closed-loop Neural Stimulation with Real-time Spike Sorting](research_papers/Closed-loop_Neural_Stimulation.pdf)
+- [Exploring IoT Smart Cities](research_papers/IoT_Smart_Cities.pdf)
+- [Personalization Revisited: A Reflective Approach Helps People Better Personalize Health Services and Motivates Them To Increase Physical Activity](research_papers/Personalization_Revisited.pdf)
+- [SnapLoc: An Ultra-Fast UWB-Based Indoor Localization System for an Unlimited Number of Tags](research_papers/SnapLoc.pdf)
+- [Synthetic Sensors: Towards General-Purpose Sensing](research_papers/Synthetic_Sensors.pdf)
+- [TSM: Temporal Shift Module for Efficient Video Understanding](research_papers/TSM.pdf)
```

Comment:
I'm really confused why the diff is so large, did you commit the wrong line endings? Make sure to use UNIX-like line endings, since that's usually what we select. I would also recommend configuring your git to do that automatically.

Reply:
If you do this right, this diff should not look so large

Reply:
When I edited the README, I went and rearranged a few courses that I had previously added (namely 18-421/623 and 18-540/745), in addition to adding 18-429/729 and 18-843 to the table. I'm not sure if that's why the diff ended up being large, but I can take a look into it if it's something else that I did

Reply:
I would just make sure your line endings are right: https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/